src/index.html

```diff
@@ -17,6 +17,10 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch">
```

Comment:
Add the modifier class here 
<img width="855" alt="image" src="https://github.com/user-attachments/assets/cec99d2e-2b3f-4195-817e-5659949cbda8">
```suggestion
    <div class="stopwatch stopwatch--speed-up">
```